curriculum/js2/10.js

```diff
@@ -7,10 +7,16 @@
  */
 
 const solution = () => {
-  Array.prototype.gsForEach = function () {
-    // To get the actual array, use this
-  };
-};
+  Array.prototype.gsForEach = function (callback, idx = 0) {
+    if (!callback) {
+      return undefined;
+    }
+    if (idx == this.length) {
+      return;
+    }
+    return this.gsForEach(callback, idx + 1);
```

Comment:
woah.... your are not doing anything with the callback? If I'm using your function:

```
[1,2,3].gsForEach( (e) => console.log(e) )
```

I will not get any console logs!

Reply:
Made changes to apply callback to current idx